.github/workflows/tests.yml

```diff
@@ -34,7 +34,7 @@ jobs:
         runs-on: ubuntu-latest
         strategy:
             matrix:
-                python-version: [3.6, 3.7, 3.8]
+                python-version: [3.8, 3.9, 3.10]
```

Comment:
```suggestion
                python-version: ["3.8", "3.9", "3.10"]
```

Reply:
@AndreyMZ Good catch. Merged suggestion and re-requested review